### PR TITLE
Kill html desc -WIP

### PR DESCRIPTION
**Pull request by neilLasrado**
#2577

### CODE REVIEW

**Comment by rmehta** [comment]
patch?

**Comment by neilLasrado** [comment]
WIP....

File: erpnext/public/js/transaction.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ erpnext.TransactionController = erpnext.stock.StockController.extend({
 
 	item_code: function(doc, cdt, cdn) {
 		var me = this;
+		var d = locals[cdt][cdn];
```
Comment (nabinhait) [inline erpnext/public/js/transaction.js]: not required

File: erpnext/patches/v5_0/update_item_desc_and_image.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2013, Web Notes Technologies Pvt. Ltd. and Contributors
+# License: GNU General Public License v3. See license.txt
+
+import frappe
+from frappe.website.utils import find_first_image
+import re
+
+def execute():
+	dt_list= ["Purchase Order Item","Supplier Quotation Item", "BOM", "Bom Explosion Item" , \
+	"Bom Item", "Opportunity Item" , "Quotation Item" , "Sales Order Item" , "Delivery Note Item" , \
```
Comment (nabinhait) [inline erpnext/patches/v5_0/update_item_desc_and_image.py]: Fix case for BOM Item, BOM Explosion Item

**Pull request merged**